addons/website_slides/__manifest__.py

```diff
@@ -19,11 +19,17 @@
 """,
     'depends': ['website', 'website_mail'],
     'data': [
+        'views/assets.xml',
         'views/res_config_settings_views.xml',
-        'views/website_slides.xml',
-        'views/website_slides_embed.xml',
-        'views/website_slides_backend.xml',
-        'data/website_slides_data.xml',
+        'views/website_slides_templates.xml',
+        'views/website_slides_embed_templates.xml',
+        'views/website_slides_slide_views.xml',
```

Comment:
Even if module name is website slides, model are named slide.channel and slide.slide. View files should therefore be slide_channel_views.xml, slide_slides_views.xml, ... Same for data or other files (mail_data.xml, website_data.xml, ...).

Website_slide_* prefix should be used whenever no main model applies to the files content. For example, a generic templates files for frontend: website_slides_templates.xml .